universe/node/api_profile.go

```diff
@@ -10,6 +11,10 @@ import (
 	"github.com/momentum-xyz/ubercontroller/utils"
 )
 
+type Form struct {
+	File *multipart.FileHeader `form:"file" binding:"required"`
+}
```

Comment:
what kind of Form is it? this type is visible for all node package

Reply:
woops that wasn't supposed to be in there, also removed the commented function below